index.js

```diff
@@ -31,7 +31,10 @@ exports.searchFirestore = functions.firestore
             var userCollection = firestore.collection(collectionName);
             if (where) {
                 where.forEach(function(subquery) {
-                    userCollection = userCollection.where(subquery.field, subquery.operator, subquery.val);
+                    if (subquery.val)
+                        userCollection = userCollection.where(subquery.field, subquery.operator, subquery.val);
+                    else
+                        userCollection = userCollection.where(subquery.field, subquery.operator, subquery.value);
```

Comment:
What if `subquery.value` also doesn't exist?  This would probably fail.

Maybe we could check if the required parameters have been added when we obtain them. If they weren't, we can throw an exception so that the catch block is called and the error is logged.